docs/source/releases/v1_11_5a0.rst

```diff
@@ -17,6 +17,22 @@ Version 1.11.5a0 (2023-09-27)
 * Update "update_this_release_note" to v1_11_5a0.rst
 * Fix Errors from Flake8 Linting
 * Remove flake8_docstring_only option from check_code.sh workflow call
+* Fixed from scipy.ndimage.interpolation import zoom deprecation warning
+
+Bug Fixes
+=========
+
+Fixed from scipy.ndimage.interpolation import zoom deprecation warning
+----------------------------------------------------------------------
+
+The scipy.ndimage.interpolation library was deprecated as of scipy v1.2 (Dec 2018), 
```

Comment:
Is it correct practice to add scipy>=1.2 to pyrpoject.toml?  I'm ok without adding a 5 year old version to pyproject.toml if others are.

Reply:
I went ahead and added scipy > 1.2 to pyproject.toml.  Resolve if you are comfortable with this, or I can back it out.